terraform/modules/logging/alarms/dropped-av-sns/alarms.tf

```diff
@@ -0,0 +1,21 @@
+resource "aws_cloudwatch_metric_alarm" "status_code_alarm" {
+  alarm_name        = "${var.environment}-dropped-antivirus-sns"
+  alarm_description = "Alarms on failure to inform the ${var.environment} AV API that a file has uploaded and needs to be scanned."
+
+  // Metric
+  namespace   = "DM-SNS"
+  metric_name = "${var.environment}-dropped-antivirus-sns"
+
+  // For for every 60 seconds
+  evaluation_periods = "1"
+  period             = "60"
+
+  // If totals 1 or higher
+  statistic           = "Sum"
+  comparison_operator = "GreaterThanOrEqualToThreshold"
+  threshold           = "1"
+
+  // Email slack
+  alarm_actions = ["${var.alarm_email_topic_arn}"]
```

Comment:
The missing logs alarm has an empty `insufficient_data_actions` in this section, do we need it here?

Reply:
Na, we don't need i there either to be fair.
You have reminded me to put a treat_missing_data value on these though. We don't care if data is missing because we have the `missing logs` alarms

Reply:
Have removed it and added missing data actions in another commit